open-authoring/PRPlugin.tsx

```diff
@@ -29,12 +34,24 @@ export const PRPlugin = (
 function PullRequestButton({
   baseRepoFullName,
   forkRepoFullName,
+  sendErrorToModal
 }: PullRequestButtonOptions) {
+
+  const open = async () => {
+    if (await isGithubTokenValid()) {
+      setOpened(p => !p)
```

Comment:
```suggestion
      setOpened(true)
```
Or is the behaviour is a toggle, the function name should change